### ISSUE TITLE
Is there a set of minimal formatting guidelines we follow?

### ISSUE THREAD

**Issue by manuel-delverme** [gehring/fax#23]

**Comment by manuel-delverme** [comment]
https://www.python.org/dev/peps/pep-0008/
is the standard and well known, i usually avoid the 80 characters limitation because monitors are bigger nowadays but that just a detail.

**Comment by pierrelux** [comment]
Adding @NeilGirdhar for advice here

**Comment by NeilGirdhar** [comment]
I agree with @manuel-delverme.  I also follow PEP8 except for the 80 character limit.  I'm using 100 characters now.  Now that I try to use type annotations, I find 80 characters to cause excessive line-wrapping.

One suggestion is to use [isort](https://github.com/timothycrosley/isort).  (Run `pip install isort`, and then `isort .`) This keeps your imports clean.  When pylint is updated (hopefully in the next month), then I would run that too.  You can steal the configurations for these tools (isort's configuration is in `pyproject.toml`, pylint has its own `pylintrc`) from my tjax project.

**Comment by pierrelux** [comment]
@gehring can you run isort/pylint for you refactor #26 ?

**Comment by gehring** [comment]
I'm planning on setting up proper CI. Part of this will be moving to poetry and cleaning up the code such that it follows some easy to check standard (e.g., flake8, isort). Let's keep this change for a follow up PR. I would like to wrap up #26.
